build/style_test.go

```diff
@@ -294,7 +355,17 @@ func TestStyle(t *testing.T) {
 
 	t.Run("TestProtoMarshal", func(t *testing.T) {
 		t.Parallel()
-		cmd, stderr, filter, err := dirCmd(pkg.Dir, "git", "grep", "-nE", `\.Marshal\([^)]+\)`, "--", "*.go")
+		cmd, stderr, filter, err := dirCmd(
+			pkg.Dir,
+			"git",
+			"grep",
+			"-nE",
+			`\.Marshal\(`,
+			"--",
+			"*.go",
+			":!util/protoutil/marshal.go",
+			":!settings/settings_test.go",
+		)
```

Comment:
This pattern really seems to deserve a `grepExcluding(`regexpattern`, "file", "file2", ...)` function, but this is an improvement as is so no need to hold up this PR.

Reply:
Eh, I'm not sure I agree that there's much benefit to the indirection. Feel free to follow up if you feel strongly enough.